src/menu/src/MenuOption.js

```diff
@@ -1,12 +1,19 @@
 import React, { memo, useCallback } from 'react'
 import PropTypes from 'prop-types'
+import useStyleConfig from '../../hooks/use-style-config'
 import { TickIcon } from '../../icons'
 import { Pane } from '../../layers'
-import { useTheme } from '../../theme'
+import { pseudoSelectors } from '../../table/src/TableRow'
 import { Text } from '../../typography'
 
 const noop = () => {}
 
+const internalStyles = {
+  display: 'flex',
+  alignItems: 'center',
+  height: 40,
```

Comment:
Should this height be in the theme?

Reply:
I think i could move it out of Menu option. but for table row it should